pkg/rehearse/jobs.go

```diff
@@ -220,6 +222,8 @@ func NewExecutor(toBeRehearsed map[string][]prowconfig.Presubmit, prNumber int,
 	dryRun bool, loggers Loggers, pjclient pj.ProwJobInterface) *Executor {
 	rehearsals := configureRehearsalJobs(toBeRehearsed, prRepo, prNumber, loggers)
```

Comment:
Since we add more stuff to the constructor, I am starting thinking that it's better to export this function and call it from `main` and pass the rehearsals to the constructor directly. Then we don't have to include the `RehearsalJobCount` in the executor and we can make the check in main without creating the `executor`.

Reply:
I think when we have an extra layer already, let's use it to some few details.